static/lang/re-en.json

```diff
@@ -2325,6 +2325,10 @@
                     "Note": "The creature is @UUID[Compendium.pf2e.conditionitems.Item.xYTAsEpcJE1Ccni3]{Slowed 1} until the end of your next turn unless it succeeds at a @Check[type:fortitude|dc:resolve(@actor.attributes.spellDC.value)] save against your spell DC."
                 }
             },
+            "Light": {
+                "ConeLabel": "Cone",
```

Comment:
It's unfortunate that these are similar to the ones for MechanicalTorch, but figured we should have them in a more general place.

Reply:
Why not just `PF2E.AreaTypeCone`?

Reply:
That could work, and `PF2E.AreaTypeEmanation` for the other?

Reply:
Yeah. Even though technically an emanation and a radius are slightly different, the difference isn't enough to quibble about for a minor item.

Reply:
Sounds good. I'll point Mechanical Torch at these strings too.